### ISSUE TITLE
[pip] ConfigParser bug in setup.py

### ISSUE THREAD

**Issue by jhunkeler** [astropy/montage-wrapper#43]
Hi @astrofrog 

The 0.9.8 release on pypi cannot be built via pip using python3.5. Is it possible for you to make a release of what is on master?

```
$ pip install montage_wrapper 
 Collecting montage_wrapper 
   Using cached montage-wrapper-0.9.8.tar.gz 
     Complete output from command python setup.py egg_info: 
     git submodule command failed unexpectedly: 
     fatal: Not a git repository (or any of the parent directories): .git 
      
     /Users/ojones/anaconda/lib/python3.5/site-packages/setuptools-23.0.0-py3.5.egg/pkg_resources/__init__.py:187: RuntimeWarning: You have iterated over the result of pkg_resources.parse_version. This is a legacy behavior which is inconsistent with the new version class introduced in setuptools 8.0. In most cases, conversion to a tuple is unnecessary. For comparison of versions, sort the Version instances directly. If you have another use case requiring the tuple, please file a bug with the setuptools project describing that need. 
     /Users/ojones/anaconda/lib/python3.5/site-packages/matplotlib/font_manager.py:273: UserWarning: Matplotlib is building the font cache using fc-list. This may take a moment. 
       warnings.warn('Matplotlib is building the font cache using fc-list. This may take a moment.') 
     /Users/ojones/anaconda/lib/python3.5/site-packages/matplotlib/font_manager.py:273: UserWarning: Matplotlib is building the font cache using fc-list. This may take a moment. 
       warnings.warn('Matplotlib is building the font cache using fc-list. This may take a moment.') 
     Traceback (most recent call last): 
       File "<string>", line 1, in <module> 
       File "/private/var/folders/2l/v4cs8jdx1rd0rzrykclw77n80001dy/T/pip-build-3tofr4b8/montage-wrapper/setup.py", line 25, in <module> 
         conf = config.ConfigParser() 
     AttributeError: module 'distutils.config' has no attribute 'ConfigParser' 
      
     ---------------------------------------- 
 Command "python setup.py egg_info" failed with error code 1 in /private/var/folders/2l/v4cs8jdx1rd0rzrykclw77n80001dy/T/pip-build-3tofr4b8/montage-wrapper/ 
```

**Comment by blakut** [comment]
I had the same error message, what i did to go around it was to install directly from git:

`pip install git+https://github.com/astropy/montage-wrapper.git`

Python 3.6.1 tho.

**Comment by astrofrog** [comment]
@jhunkeler @blakut - apologies for the long delay, I've now released 0.9.9 which fixes compatibility with Python 3.6
